cms/src/components/content/EditContent.vue

```diff
@@ -125,6 +128,7 @@ const createTranslation = (language: LanguageDto) => {
         title: `Translation for ${language.name}`,
         slug: "",
         parentTags: [],
+        publishDateVisible: true,
```

Comment:
No need to set the `publishDateVisible` field on the ContentDto documents as this is done in the API.